crev-lib/src/local.rs

```diff
@@ -386,8 +390,16 @@ impl Local {
         })
     }
 
+    fn ensure_proofs_root_exists(&self) -> Result<()> {
+        let proofs_dir = self.user_proofs_path();
+        if !proofs_dir.exists() {
```

Comment:
Note: there's really no point in checking. `create_dir_all` is written in opportunistic way and if the dir exists, it will take the same amount of time as checking itself.